test/system/src/main/java/io/pravega/test/system/framework/DockerRemoteSequential.java

```
@@ -0,0 +1,162 @@
+        throw new org.apache.commons.lang.NotImplementedException("Stop Execution is not used for Remote sequential execution");
+    }
+
+    private CompletableFuture<Void> waitForJobCompletion(final String containerName) {
+        return FutureHelpers.loop(() -> isTestRunning(containerName),
+                () -> FutureHelpers.delayedFuture(Duration.ofSeconds(3), executorService),
+                executorService);
+    }
+
+    private boolean isTestRunning(String containerName) {
+        boolean value = false;
+        try {
+            //list the service with filter 'serviceName'
+            List<Container> containers = CLIENT.listContainers(DockerClient.ListContainersFilterParam.withStatusRunning());
+            for (int i = 0; i < containers.size(); i++) {
+                ContainerInfo info = CLIENT.inspectContainer(containers.get(i).id());
+                if (info.name().equals(containerName)) {
+                    value = true;
+                    break;
+                }
+            }
+        } catch (DockerException | InterruptedException e) {
+            log.error("unable to list docker services {}", e);
+        }
+        return value;
+    }
+
+    private String startTest(String containerName, String className, String methodName) {
+
+        try {
+            CLIENT.pull(IMAGE);
+
+            ContainerCreation containerCreation = CLIENT.createContainer(setContainerConfig(methodName, className), containerName);
+            assertFalse(containerCreation.id().toString().equals(null));
+
+              id = containerCreation.id();
+
+            final Path dockerDirectory = Paths.get(System.getProperty("user.dir"));
+            try {
+                CLIENT.copyToContainer(dockerDirectory, id, "/data");
+            } catch (IOException | DockerException | InterruptedException e) {
+                log.error("Exception while copying test jar to the container {}", e);
```

we should not suppress exceptions here and below.

---

Failing the test , if not able to start container or copying the test jar to container.